src/content/admin/cloud-credentials/index.mdx

```diff
@@ -7,7 +7,7 @@ sidebar_label: Cloud Credentials
 import Image from "@theme/Image";
 
 In the Admin Dashboard, you can configure Cloud Credentials for accessing your cloud account.
-Cloud Credentials are automatically used by Okteto in your deploy, destroy, and test commands, eliminating the need for developers to manage static credentials.
+Cloud Credentials are automatically used by Okteto in your deploy, destroy, and test commands, eliminating the need of manage static credentials.
```

Comment:
I’ve removed the reference to the developer. Permission management typically falls under the responsibility of the security or platform teams, and I believe it’s more appropriate to direct the message towards Cindy and/or Cindy's team taking care of that task.